troubleshooting.rst

```diff
@@ -0,0 +1,390 @@
+===============
+Troubleshooting
+===============
```

Comment:
@JustinAzoff  - not sure how easy this is for you to read, but if something comes to mind that could be generally valuable please state.

There hasn't been a place where we document basic `perf` or `jemalloc` usage/debugging and maybe Zeek docs is a bit of an odd place, but it's better than no place and tribe knowledge.